server/routes/students.js

```diff
@@ -0,0 +1,62 @@
+const router = require('express').Router();
+const { Student, Test } = require('../db/models');
+
+// GET /api/students
+router.get('/', async (req, res, next) => {
+  try {
+    const students = await Student.findAll({
+      include: { all: true }
+    });
+    res.json(students);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// GET /api/students/:id
+router.get('/:id', async (req, res, next) => {
+  try {
+    const student = await Student.findByPk(req.params.id);
+    if (student) res.json(student);
+    else res.sendStatus(404);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// PUT /api/students/:id
+router.put('/:id', async (req, res, next) => {
+  try {
+    const { firstName, lastName, email } = req.body;
+    const [rowsAffected, affectedRows] = await Student.update(
```

Comment:
IMO we should move away from the model update/destroy and use the instance ones instead. Works better with hooks, but also, we're not operating on the whole table so we _should_ be using the instance method here. 

So we could change this to a `Student.findByPk` followed by a `studentInstance.update`

Reply:
The instance update I believe will return the updated instance without needing to include "returning: true", and the result returned is easier to digest IMO (it's just 1 value, not something that needs to be destructured)

Reply:
Word up. "Updated"! ;)